PrimeService.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Xunit;
+
+namespace PrimeService.Tests

```

Comment:
These tests have been added to ensure the setup actually works. The file will be changed when adding the first real tests.